plugin/trino-mongodb/src/test/java/io/trino/plugin/mongodb/BaseMongoDistributedQueries.java

```diff
@@ -78,10 +76,6 @@ protected TestTable createTableWithDefaultColumns()
     public void testColumnName(String columnName)
     {
         if (columnName.equals("a.dot")) {
-            // TODO (https://github.com/trinodb/trino/issues/3460)
-            assertThatThrownBy(() -> super.testColumnName(columnName))
-                    .hasStackTraceContaining("TableWriterOperator") // during INSERT
-                    .hasMessage("Invalid BSON field name a.dot");
```

Comment:
You need to throw an exception explicitly since the driver's behavior is changed during version upgrade.

Reply:
What exception would you like to be thrown